<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buscar</class>
 <widget class="QWidget" name="Buscar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_22">
     <item>
      <layout class="QHBoxLayout" name="Vuelo">
       <item>
        <layout class="QVBoxLayout" name="CiudadO">
         <item>
          <widget class="QLabel" name="LCd_origen">
           <property name="text">
            <string>Ciudad de Origen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Cd_origen">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>México - D.F.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>México - Guadalajara</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canada - Montreal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USA - New York</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USA - Washington</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="CiudadD">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Ciudad de Destino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Cd_destino">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>México - D.F.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>México - Guadalajara</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canada - Montreal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USA - New York</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USA - Washington</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Salidas">
         <item>
          <widget class="QLabel" name="LSalida">
           <property name="text">
            <string>Salida</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="Salida"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="egresos">
         <item>
          <widget class="QLabel" name="LRegreso">
           <property name="text">
            <string>Regreso</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="Regreso"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="TablaVuelos">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Origen</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Destino</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Salida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Regreso</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Asientos</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="EliminarVuelo">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Nombre"/>
     </item>
     <item>
      <widget class="QTableWidget" name="TablaPasajeros">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Vuelo</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
